jwql/tests/test_data_containers.py

```diff
@@ -159,25 +159,27 @@ def test_get_available_suffixes(untracked, input_suffixes, expected):
 
 def test_get_current_flagged_anomalies(mocker):
     # get a sample query group with 2 files
-    rootname = 'jw02589006001_04101_00001-seg001'
-    instrument = 'NIRSpec'
+    # TODO update this test to use the new django models
```

Comment:
Are you waiting to update this in a future PR?

Reply:
Yup, I tagged you elsewhere regarding it.  We'll need to implement Django Model testing which should be its own issue.

Reply:
This test was one of my new additions, to test the group anomaly behavior and fill out coverage in data_containers.  I can refactor it later.